java/registry/src/main/java/io/opensaber/registry/service/DecryptionHelper.java

```diff
@@ -0,0 +1,34 @@
+package io.opensaber.registry.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import io.opensaber.registry.exception.EncryptionException;
+import io.opensaber.registry.service.EncryptionService;
+import io.opensaber.registry.util.Definition;
+import io.opensaber.registry.util.DefinitionsManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class DecryptionHelper extends EncryptionHelper{
```

Comment:
If you need some methods from EncryptionHelper, please elevate it to base class...dont extend. It can confuse.

Reply:
Created PrivateField class for common methods and removed this extends to EncryptionHelper.